ee/tabby-db/migrations/0010_server-settings.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE server_setting (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    security_disable_clientside_telemetry BOOLEAN NOT NULL DEFAULT FALSE,
+    network_external_url STRING NOT NULL DEFAULT 'http://localhost:8080',
+    security_allowed_register_domains STRING
```

Comment:
```suggestion
    security_allowed_register_domain_list STRING
```

preferring use `domain_list`, in general we don't want to have plural in field names.